### PR TITLE
Review of Redux RTK and TypeScript Notes 

### PR DESCRIPTION
**Pull request by zacjones93**
The notes are looking good. 

I'm seeing a couple of patterns that should be addressed. 

Any time we link to docs or elsewhere, we need to give a concise summary of what is being linked to. Saying 'more on `:method-name`' isn't super helpful to the learner. Give them a reason to click that link.

The other one, is there a reason for breaking up codeblocks a lot? Could look a little cleaner doing slightly larger blocks. In addition, provide a little more context in the code block. I'll add more to that in the review down below but typically (if it's not obnoxious) you can provide the whole function or render call if there isn't too much code there

Could you work through the notes with this feedback in mind?

### CODE REVIEW

File: modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md [hunk 1]
```diff
@@ -4,16 +4,15 @@
 
 <TimeStamp start="0:02" end="0:10">
 
-```
-npm install react-redux
-npm install @reduxjs/toolkit
+```bash
+npm install react-redux @reduxjs/toolkit
 ```
```
Comment (zacjones93) [inline modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md:9]: for people who want to copy paste the install, you can do multiple packages at once by comma separating them

File: modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md [hunk 2]
```diff
@@ -4,16 +4,15 @@
...
 
 </TimeStamp>
 
 <TimeStamp start="0:16" end="0:31">
 
-More on [configureStore](https://redux-toolkit.js.org/api/configureStore)
+There are a number of options and parameters you can set for [configureStore in the docs](https://redux-toolkit.js.org/api/configureStore)
```
Comment (zacjones93) [inline modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md:15]: provide a nice summary of the links

File: modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md [hunk 3]
```diff
@@ -37,9 +36,14 @@ import { store } from "./app/store";
...
-</Provider>
+ReactDOM.render(
+  <React.StricMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StricMode>,
+  document.getElementById("root")
+);
 ```
```
Comment (zacjones93) [inline modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md:47]: gives more context in the note when it doesn't add to much noise

File: modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/04_react-splitting-our-redux-store-into-multiple-slices-using-the-createslice-method.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 [Video link](https://www.egghead.io/lessons/react-splitting-our-redux-store-into-multiple-slices-using-the-createslice-method?pl=modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8)
 
-<TimeStamp start="0:22" end="0:35">
+<TimeStamp start="0:31" end="0:49">
```
Comment (zacjones93) [inline modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/04_react-splitting-our-redux-store-into-multiple-slices-using-the-createslice-method.md:5]: these note timestamps on this one were all off 🤔  not sure why

File: modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/05_react-creating-a-rootstate-type-and-typed-hooks-for-type-aware-redux-interactions.md [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 <TimeStamp start="0:02" end="0:18">
 
-More on [dispatch](https://redux.js.org/usage/usage-with-typescript#define-root-state-and-dispatch-types)
+[dispatch](https://redux.js.org/tutorials/fundamentals/part-2-concepts-data-flow#dispatch) is how you update state in your store. You'll pass dispatch an action object.
```
Comment (zacjones93) [inline modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/05_react-creating-a-rootstate-type-and-typed-hooks-for-type-aware-redux-interactions.md:7]: another example of adding more context, think progressive summarization. Why is the link useful?

**Comment by lsminter** [comment]
Yeah these are all good edits. Not sure about the timestamp on those one notes. I could have just miss remembered the time when I was writing it down.

**Pull request merged**